inspector.ipxe

```diff
@@ -4,6 +4,6 @@
 echo In inspector.ipxe
 imgfree
 # NOTE(dtantsur): keep inspection kernel params in [mdns]params in ironic-inspector-image
-kernel --timeout 60000 http://IRONIC_IP:HTTP_PORT/images/ironic-python-agent.kernel ipa-inspection-callback-url=http://IRONIC_IP:5050/v1/continue ipa-inspection-collectors=default,extra-hardware,logs systemd.journald.forward_to_console=yes BOOTIF=${mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs || goto retry_boot
+kernel --timeout 60000 http://IRONIC_IP:HTTP_PORT/images/ironic-python-agent.kernel ipa-inspection-callback-url=http://IRONIC_IP:5050/v1/continue ipa-api-url=http://IRONIC_IP:6385 ipa-inspection-collectors=default,extra-hardware,logs systemd.journald.forward_to_console=yes BOOTIF=${mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs || goto retry_boot
```

Comment:
You'll want to use `$IRONIC_URL_HOST` so IPv6 addresses are bracketed

Reply:
What's weird is not we're not using that variable for the other URL's.  I'm wondering how this worked -- is this file even used?

Reply:
We've discovered that this file is not used for normal (non-discovery) inspection. I'll copy my explanation from slack:

```
16:43 so, we have two modes of starting inspection: independent via inspector.ipxe or "managed" - via ironic, the same way as cleaning and deployment starts
16:43 ⤷ discovery always goes through the first path, but standalone inspection usually goes through the second (the change was not in Train but has been backported downstream).
16:43 ⤷ the second path does not use configuration in inspector.ipxe, resulting in what you observe
16:45 ⤷ we need two things: fix ironic.conf to provide inspection collectors and fix ironic itself to provide ipa-api-url when fast_track is enabled
16:46 ⤷ (actually, the latter can also be worked around via the image)
```

Reply:
Also note that IRONIC_IP here is replaced by `$IRONIC_URL_HOST` in `runhttp.sh`